models/bamf_mr_brain_tumor/meta.json

```diff
@@ -0,0 +1,206 @@
+    "license": {
+      "code": "MIT",
+      "weights": "CC BY-NC 4.0"
+    },
+    "publications": [
+      {
+        "title": "The AIMI Initiative: AI-Generated Annotations in IDC Collections",
+        "uri": "https://arxiv.org/abs/2310.14897"
+      }
+    ],
+    "github": "https://github.com/bamf-health/mhub-models/tree/bamf_mr_brain_tumor"
+  },
+  "info": {
+    "use": {
+      "title": "Intended Use",
+      "text": "This model is intended to perform tumor segmentation in Brain MRI scans. The model has been trained and tested on scans aquired during clinical care of patients, so it might not be suited for a healthy population. The generalization capabilities of the model on a range of ages, genders, and ethnicities are unknown. For detailed information on the training set design, please refer to reference section in the training section"
+    },
+    "analyses": {
+      "title": "Quantitative Analyses",
+      "text": "The model's performance was assessed using the Dice Coefficient and Normalized Surface Distance (NSD) with tolerance 7mm. The model was used to segment cases from the IDC collection UPENN-GBM [1]. Seven of those cases were reviewed and corrected by a board-certified radiologist and a non-expert. The analysis is published here [2]",
+      "tables": [
+        {
+          "label": "Label-wise metrics Dice score between AI derived and manually corrected annotations",
+          "entries": {
+            "Whole Tumor": "0.98±0.07",
+            "Enhancing Tumor": "0.95±0.13",
+            "Non enhancing Tumor": "0.97±0.08"
+          }
+        },
+        {
+          "label": "Label-wise metrics 95% Hausdorff Distance between AI derived and manually corrected annotations",
+          "entries": {
+            "Whole Tumor": "6.88±0.34",
+            "Enhancing Tumor": "6.57±0.24",
+            "Non enhancing Tumor": "0.42±1.05"
+          }
+        },
+        {
+          "label": "Label-wise metrics NSD score between AI derived and manually corrected annotations",
+          "entries": {
+            "Whole Tumor": "0.98±0.04",
+            "Enhancing Tumor": "0.99±0.03",
+            "Non enhancing Tumor": "0.97±0.09"
+          }
+        }
+      ],
+      "references": [
+        {
+          "label": "UPENN-GBM",
+          "uri": "https://www.cancerimagingarchive.net/collection/upenn-gbm/"
+        },
+        {
+          "label": "The AIMI Initiative: AI-Generated Annotations for Imaging Data Commons Collections",
+          "uri": "https://arxiv.org/abs/2310.14897"
+        }
+      ]
+    },
+    "evaluation": {
+      "title": "Evaluation Data",
+      "text": "The model was used to segment cases from the IDC [1] collection UPENN-GBM [1]. Approximately 55 cases were randomly selected to be reviewed and corrected by a board-certified radiologist. The model predictions, and radiologist corrections are published",
```

Comment:
> Approximately 55 cases were randomly selected to be reviewed and corrected by a board-certified radiologist

What does *Approximately* mean in this context?

Reply:
Missed to include Approximately 10% of the data. I have updated it now